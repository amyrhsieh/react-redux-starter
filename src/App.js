import React from 'react';
import Header from './components/Header';
import Main from './components/Main';
import Footer from './components/Footer';
// import 'normalize.css';
//
// import "styles/base/_main.scss";
// import "styles/base/_common.scss";
// import "styles/base/_reset.scss";
// import styles from "./app.scss";

const App = () => (
  <React.Fragment>
    <Header />
    <Main />
    <Footer />
  </React.Fragment>
);

export default App;
