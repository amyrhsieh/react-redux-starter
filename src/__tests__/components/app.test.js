import React from 'react';
import renderer from 'react-test-renderer';
import App from 'Src/App.js';
import { shallow } from 'enzyme';

describe('App (Snapshot)', () => {
  it('App component renders App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
