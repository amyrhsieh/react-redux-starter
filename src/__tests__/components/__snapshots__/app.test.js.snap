// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App (Snapshot) App component renders App 1`] = `
<Fragment>
  <Header />
  <Main />
  <Footer />
</Fragment>
`;
